Content/LeagueSandbox-Scripts/Characters/Ezreal/E.cs

```diff
@@ -82,6 +90,26 @@ public void OnSpellPostChannel(ISpell spell)
 
         public void OnUpdate(float diff)
         {
+            if (_owner == null || _sector == null) 
```

Comment:
Move all of this into a `TargetExecute` function. Refer to [Annie W](https://github.com/LeagueSandbox/GameServer/blob/indev/Content/LeagueSandbox-Scripts/Characters/Annie/W.cs#L26) implementation.

Reply:
🔨 https://github.com/LeagueSandbox/GameServer/pull/1459/commits/8424234e6f2418da5caf45a5c57d7a4b7761e46d